frontend/src/app/home/containers/Home.js

```diff
@@ -121,11 +121,66 @@ export default class Home extends React.Component {
           });
           return null;
         });
+
+
+  }
+
+  sellStock(stock, quantity, price){
+    let owned = 3;
+    if (quantity === ''){
+      alert("Can't sell zero shares!")
+    }
+    else if (quantity > owned){
+      alert("Can't sell more shares than you own!")
+    }
+    else {
+      alert(`Sold ${quantity}x ${stock} shares!`)
+    }
+
+  }
+
+
+  postFeed() {
```

Comment:
I think this has been supplanted by another function.

Reply:
Will remove